src/linter-jscs.js

```diff
@@ -97,6 +97,19 @@ export default class LinterJSCS {
         }
       });
     }));
+
+    this.subscriptions.add(atom.commands.add('atom-text-editor', {
+      'linter-jscs:fix-file': () => {
+        const textEditor = atom.workspace.getActiveTextEditor();
+
+        if (!textEditor || textEditor.isModified()) {
```

Comment:
Since `jscs` can fix a string the check as to whether it has been modified isn't necessary here and can be removed.

Also note that the message below should then be changed to something along the lines of "Invalid TextEditor received, aborting.". (Which will only trigger if there is a bug in Atom anyway...)